.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will launch at 00:00 every Sunday
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+name: Launch Scala Steward
```

Comment:
Can this line be moved to the top of the file (like the other one)?

Reply:
sure